definitions/features/downstream.rb

```diff
@@ -55,22 +55,39 @@ def rhsm_refresh
   def rh_repos(sat_version)
     sat_version = version(sat_version)
     rh_version_major = ForemanMaintain::Utils::Facter.os_major_release
-    sat_version_full = "#{sat_version.major}.#{sat_version.minor}"
 
+    rh_repos = main_rh_repos(rh_version_major)
+
+    rh_repos.concat(sat_and_tools_repos(rh_version_major, sat_version))
+
+    rh_repos << 'rhel-7-server-ansible-2-rpms' if sat_version.to_s == '6.4'
+
+    if current_minor_version == '6.3' && (
```

Comment:
@kgaikwad @iNecas The PR looks good to me...except one question while upgrading from sat 6.3 puppet 4 to sat 6.4 we are enabling puppet 4 repo which is not needed also i am not sure will it affect anything or not...shall we add one more condition here that to version != 6.4 ?

Reply:
I have updated the PR with this change. Please have a look!